Tools/Scripts/run-api-tests

```diff
@@ -145,6 +145,8 @@ def parse_args(args):
                              help='Force all tests to run in parallel (without grouping shards)'),
         optparse.make_option('--additional-env-var', type='string', action='append', default=[],
                              help='Passes that environment variable to the tests (--additional-env-var=NAME=VALUE)'),
+        optparse.make_option('--test-parallel-safety-with', type='string', action='append', default=[],
```

Comment:
You may want to drop `-with` from the option name, as the rest of your comments and variables are `test_parallel_safety` sans `with`.